conftest.py

```diff
@@ -333,3 +379,38 @@ def unix2dos():
 def jsonschema_validator():
     """Provide the standard JSON schema validator."""
     return jsonschema.Draft4Validator
+
+
+@pytest.fixture(scope="session")
+def gitattributes():
```

Comment:
This fixture can be removed, I think? And, if so, I think also the `os` import at the top.